specification/08-attestation.adoc

```diff
@@ -1,2 +1,117 @@
 [[attestation]]
 == Device Attestation
+
+There are two attestation stages involved in the CoVE-IO architecture.
+- TVM attests the device
```

Comment:
Nit: You need a breaking line here or else it won't be nicely formatted into 2 separare points.

Reply:
yes. Done